src/v2/view-builder/views/ov/ChallengeOktaVerifyTotpView.js

```diff
@@ -13,6 +15,38 @@ const Body = BaseForm.extend(Object.assign(
     save() {
       return loc('mfa.challenge.verify', 'login');
     },
+
+    initialize: function() {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      this.listenTo(this.options.appState, 'afterError', (convertedErrors) => {
```

Comment:
Isnt afterError  triggered only after form is rendered and then an error occurs on submit.

Reply:
`showFormErrors` trigger `error` that trigger `afterError`.. It is on submit but doesn't do re-render

Reply:
what I was getting at was that we might not need the timeout here since the afterError would be triggered only on error during form submit at which the form is already rendered.